### PR TITLE
fix: center and focus not being allowed in config

### PR DESCRIPTION
**Pull request by FabianLars**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by FabianLars** [comment]
The only problem i have is that changing focus via the config doesn't make a difference at all (at least on Windows).

File: core/tauri-runtime-wry/src/lib.rs [hunk 1]
```diff
@@ -442,6 +442,10 @@ impl WindowBuilder for WindowBuilderWrapper {
       window = window.position(x, y);
     }
 
+    if config.center {
```
Comment (FabianLars) [inline core/tauri-runtime-wry/src/lib.rs:445]: This way center overwrites x and y. I don't know if it should do that tho.

Comment (FabianLars) [inline core/tauri-runtime-wry/src/lib.rs:445]: This way center overwrites x and y. I don't know if it should do that tho.

File: tooling/cli.rs/config_definition.rs [hunk 2]
```diff
@@ -292,6 +295,9 @@ pub struct WindowConfig {
   /// Whether the window starts as fullscreen or not.
   #[serde(default)]
   pub fullscreen: bool,
+  /// Whether the window will be initially hidden or focused.
+  #[serde(default = "default_focus")]
+  pub focus: bool,
```
Comment (FabianLars) [inline tooling/cli.rs/config_definition.rs:300]: Changing this in tauri.conf.json doesn't seem to make a difference (at least on Windows).

Comment (amrbashir) [reply tooling/cli.rs/config_definition.rs:300]: We decided to remove `focus` from `WindowBuilder` and `WindowAttributes` at Tao, so if tauri wants to support this, they would have to explicitly call `window.set_focus()`.

Comment (FabianLars) [reply tooling/cli.rs/config_definition.rs:300]: Should i remove the focus stuff again? -> Make this pr only about the center thingy?

Comment (amrbashir) [reply tooling/cli.rs/config_definition.rs:300]: It is @lucasfernog decision.

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**